amundsen_application/static/js/components/common/BadgeList/index.tsx

```diff
@@ -58,12 +58,10 @@ const ActionableBadge: React.FC<ActionableBadgeProps> = ({
 };
 
 export class BadgeList extends React.Component<BadgeListProps> {
-  idx = 0;
-
-  handleClick = (e) => {
-    const badgeText = this.props.badges[this.idx].badge_name
-      ? this.props.badges[this.idx].badge_name
-      : this.props.badges[this.idx].tag_name;
+  handleClick = (index: number, e) => {
+    const badgeText = this.props.badges[index].badge_name
```

Comment:
Nit: We prefer destructuring the props before using ( const {badges} = this.props; )